testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-compatibility/hive.properties

```diff
@@ -0,0 +1,16 @@
+# Copy of common/hadoop/hive.properties with older connector name (versions < 359 don't have connector.name=hive)
+connector.name=hive-hadoop2
+hive.metastore.uri=thrift://hadoop-master:9083
+hive.config.resources=/docker/presto-product-tests/conf/presto/etc/hive-default-fs-site.xml
+hive.allow-add-column=true
+hive.allow-drop-column=true
+hive.allow-rename-column=true
+hive.allow-comment-table=true
+hive.allow-comment-column=true
+hive.allow-drop-table=true
+hive.allow-rename-table=true
+hive.allow-register-partition-procedure=true
+hive.metastore-cache-ttl=0s
+hive.fs.cache.max-size=10
+hive.max-partitions-per-scan=100
+hive.translate-hive-views=true
```

Comment:
are these relevant?

Reply:
This is a 1:1 copy of the catalog definition used by the "current" server. But yes, they are not "used" by the tests I added. I'll trim this down.

Reply:
Removed everything except `hive.metastore.uri` and the tests still work.